### ISSUE TITLE
Feature Engineering — deep dive into Encoding and Binning techniques

### ISSUE THREAD

**Issue by rhiriyappa** [rhiriyappa/data-science-articles#13]
# TL;DR
**Feature engineering** is the most important aspect of a data science model development. There are several categories of features in a raw dataset. Features can be text, date/time, categorical, and continuous variables. For a machine learning model, the dataset needs to be processed in the form of numerical vectors to train it using an ML algorithm.
The objective of this article is to demonstrate feature engineering techniques to transform a categorical feature into a continuous feature and vice-versa.

- **Feature Binning:** Conversion of a continuous variable to categorical.

- **Feature Encoding:** Conversion of a categorical variable to numerical features.
![image](https://user-images.githubusercontent.com/15959961/115500762-d3c08a80-a226-11eb-8b97-b2fecdc3938b.png)

# Article Link
https://towardsdatascience.com/feature-engineering-deep-dive-into-encoding-and-binning-techniques-5618d55a6b38

#Key Takeaway
